lib/hooks.js

```diff
@@ -195,7 +195,11 @@ function onSendHookRunner (functions, request, reply, payload, cb) {
     }
 
     if (i === functions.length) {
-      cb(null, request, reply, payload)
```

Comment:
@mcollina this will not decrease the performance since it can't be optimized by v8?

Reply:
If yes, would be good a benchmark for it, only to be sure how much it will decrease.

Reply:
Not anymore, try-catch is totally safe since Turbofan.